src/pages/Models/Details/ModelDetails.test.js

```diff
@@ -32,6 +32,7 @@ describe("ModelDetail Container", () => {
     );
     expect(wrapper.find("Topology").length).toBe(1);
     expect(wrapper.find(".model-details__main table").length).toBe(2);
+    expect(wrapper.find("StatusStrip")).toBeDefined();
```

Comment:
The scope of these tests are becoming very large. It would be useful to separate each aspect when one fails.